### PR TITLE
Add comment about BT.2100 in avif.h

### PR DESCRIPTION
**Pull request by y-guyon**
See https://github.com/AOMediaCodec/libavif/issues/1825.

### CODE REVIEW

File: include/avif/avif.h [hunk 1]
```diff
@@ -324,6 +324,10 @@ enum
     AVIF_COLOR_PRIMARIES_SMPTE431 = 11,
     AVIF_COLOR_PRIMARIES_SMPTE432 = 12, // DCI P3
     AVIF_COLOR_PRIMARIES_EBU3213 = 22
+
+    // BT.2100 can be achieved through AVIF_COLOR_PRIMARIES_BT2020 and
```
[the remark this answers is not in the document]
Comment (y-guyon) [reply include/avif/avif.h]: Done, PTAL

File: include/avif/avif.h [hunk 2]
```diff
@@ -336,7 +336,7 @@ enum
     // This is actually reserved, but libavif uses it as a sentinel value.
     AVIF_TRANSFER_CHARACTERISTICS_UNKNOWN = 0,
 
-    AVIF_TRANSFER_CHARACTERISTICS_BT709 = 1,
+    AVIF_TRANSFER_CHARACTERISTICS_BT709 = 1, // Same as sRGB
```
Comment (maryla-uc) [inline include/avif/avif.h:339]: remove this comment, the sRGB transfer curve is actually AVIF_TRANSFER_CHARACTERISTICS_SRGB
only the primaries match BT709

**Review by maryla-uc — APPROVED** [review]
thanks for the changes!

File: include/avif/avif.h [hunk 3]
```diff
@@ -350,9 +350,9 @@ enum
     AVIF_TRANSFER_CHARACTERISTICS_SRGB = 13,
     AVIF_TRANSFER_CHARACTERISTICS_BT2020_10BIT = 14,
     AVIF_TRANSFER_CHARACTERISTICS_BT2020_12BIT = 15,
-    AVIF_TRANSFER_CHARACTERISTICS_SMPTE2084 = 16, // PQ
+    AVIF_TRANSFER_CHARACTERISTICS_SMPTE2084 = 16, // PQ (BT.2100)
```
Comment (kmilos) [inline include/avif/avif.h:353]: "also BT.2100"

File: include/avif/avif.h [hunk 4]
```diff
@@ -350,9 +350,9 @@ enum
...
     AVIF_TRANSFER_CHARACTERISTICS_SMPTE428 = 17,
-    AVIF_TRANSFER_CHARACTERISTICS_HLG = 18
+    AVIF_TRANSFER_CHARACTERISTICS_HLG = 18 // (BT.2100)
```
[the remark this answers is not in the document]
Comment (y-guyon) [reply include/avif/avif.h]: May I know in which specification(s) this information can be looked up, if someone were to stumble on this PR in the future?

[the remark this answers is not in the document]
Comment (kmilos) [reply include/avif/avif.h]: There are tables that cover all standards/references in ITU H.273

**Pull request merged**